src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/CancellationTokenParametersMustComeLastTests.cs

```diff
@@ -218,6 +218,28 @@ class T : I
             VerifyCSharp(test, expected);
         }
 
+        [Fact, WorkItem(1491, "https://github.com/dotnet/roslyn-analyzers/issues/1491")]
+        public void NoDiagnosticOnCancellationTokenExtensionMethod()
+        {
+            var test = @"
+using System.Threading;
+static class C1
+{
+    public static void M1(this CancellationToken p1, object p2)
```

Comment:
:bulb: It would be good to add a new test where a `CancellationToken` is passed as the second argument of a 3-argument extension method (a diagnostic should be reported in this case).